woocommerce/woocommerce-android
# Introduce waiting time track capability

Summary
==========
Partially fix issues #7063 and #7062 by introducing a Waiting time tracker feature that when notified with a `start` and `end` calls, will submit the elapsed time to our Tracks with the provided analytics track. This allows us to measure how long our user waits between doing any action and seeing the result. Right now, this will be used specifically for the Orders performance project, allowing us to measure the waiting time to load the Order Details and Order List views.

The work introduced here is the Android counterpart of [this iOS one](https://github.com/woocommerce/woocommerce-ios/pull/7417)

How to Test
==========
1. Launch the app
2. Wait for the My Store to load
3. Go to the Orders section and wait for the Order list to load
4. Select any Order in that list and wait for the Order details to load
5. Verify in the Tracks logs that the following information is showing up:
<img width="579" alt="Screen Shot 2022-08-04 at 14 45 20" src="https://user-images.githubusercontent.com/5920403/182917847-9371764a-4658-40e8-a89c-2e6d8d924319.png">


 

Update release notes:
